src/Microsoft.SqlTools.ServiceLayer/Microsoft.SqlTools.ServiceLayer.csproj

```diff
@@ -21,9 +21,9 @@
 </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Data.SqlClient" Version="4.5.0" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.6.0-preview3-27014-02" />
```

Comment:
this is going to cause problems with our private SqlClient patch for the macOS KeepAlive.  If we really need to move to a new SqlClient version let's sync-up on how to build a new private build with that fix.

Reply:
we need the new SqlClient version because 4.5.0 doesn't have `AccessToken` on connections

Reply:
@MattIrv we'll need to build a private SqlClient for macOS that matches the `4.6.0-preview3-27014-02` sync point.  I'll track down where this is building from, but you'll need to include that updated binary with this PR to avoid breaking the KeepAlive scenario.  And the build scripts currently apply the patch, so whatever you reference will be overwritten in the packaged archives.

Reply:
I haven't figured out how to track down what code corresponds to that release yet but when I tested with the patched binaries everything worked

Reply:
The [nuget gallery for that package](https://www.nuget.org/packages/System.Data.SqlClient/4.6.0-preview3-27014-02) shows that it comes from commit a8b44a7 in the corefx release/2.2 branch. In your repo that's https://github.com/kburtram/corefx/commit/a8b44a739a4ffef665a02bc83ba1355e0ee945df. So if you haven't updated your release/2.2 branch since building the custom binaries then they should work 4.6.0-preview3-27014-02.